.github/workflows/test.yml

```diff
@@ -1,59 +1,34 @@
----
-    name: Test
-    on:
-      push:
-        branches:
-          - main
-      pull_request:
-    
-    jobs:
-      test:
-        strategy:
-          matrix:
-            go-version: [ 1.21.x ]
-            goarch: [ "amd64" ]
-        runs-on: ubuntu-latest
-        steps:
-        - name: Checkout code
-          uses: actions/checkout@v3
-    
-        - name: Install Go
-          uses: actions/setup-go@v3
-          with:
-            go-version: ${{ matrix.go-version }}
-          env:
-            GOARCH: ${{ matrix.goarch }}
-    
-        - name: Test
-          run: make test
-          working-directory: ./
+name: Test
+on:
+  push:
+    branches:
+      - main
+  pull_request:
 
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.21.x]
+        goarch: ["amd64"]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
 
-        - name: Archive code coverage results
-          uses: actions/upload-artifact@v3
-          with:
-            name: code-coverage-report
-            path: coverage.out
+      - name: Install Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ matrix.go-version }}
+        env:
+          GOARCH: ${{ matrix.goarch }}
 
-      sonar-cloud:
```

Comment:
I'm lacking context, why we are removing this?

Reply:
It was obsolete and not working. Fixed it now.